spec/factories/reports.rb

```diff
@@ -31,7 +33,11 @@
         analysis: true,
         literature: true }
     end
-    objects { [{ id: 4, type: 'sample' }, { id: 5, type: 'Reaction' }] }
+    objects do
+      [
+        # ActiveSupport::HashWithIndifferentAccess.new(id: 4, type: 'sample'),
```

Comment:
Is this supposed to be an empty array? What does the comment indicate?

Reply:
The comment allows ruby to access keys in the hash either by symbol :id or by string "id" . By default it is only accessible by symbol.
The empty array here is at first just a placeholder and can be populated with values if the usecases are tested.